neqo-http3/src/connection_client.rs

```diff
@@ -5562,4 +5656,311 @@ mod tests {
             assert_closed(&client, &Error::HttpSettings);
         }
     }
+
+    #[test]
+    fn response_w_1xx() {
+        let (mut client, mut server, request_stream_id) = connect_and_send_request(true);
+
+        setup_server_side_encoder(&mut client, &mut server);
+
+        let headers1xx = vec![(String::from(":status"), String::from("101"))];
```

Comment:
101 might be a bad example.  It's not valid in h3.

Should we reset streams if we get 101?

```suggestion
        let headers1xx = vec![(String::from(":status"), String::from("103"))];
```

Reply:
We should. I will open an issue.